src/test/java/page/GoogleHomePage.java

```diff
@@ -0,0 +1,36 @@
+package page;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class GoogleHomePage extends BasePage {
+
+    private static final String GOOGLE_HOMEPAGE_URL = "https://www.google.com/";
+    private WebDriver driver;
```

Comment:
You don't need here WebDriver, as you extend Base Page - you'll use it from there

Reply:
Sure, thanks. Just changed private **WebDriver driver; -> public WebDriver driver;** in BasePage class